app/src/main/res/values/styles.xml

```diff
@@ -36,14 +36,38 @@
 		<item name="android:background">@drawable/keyboard_button_selector</item>
 	</style>
 
+	<style name="ListItemIcon">
+		<item name="android:layout_marginLeft">16dp</item>
```

Comment:
Should this use `@dimen/abc_list_item_padding_horizontal_material`?

Check out https://github.com/android/platform_frameworks_support/blob/master/v7/appcompat/res/values/dimens_material.xml#L36

Maybe there's a theme or style to what you're doing here in the support library too.